### ISSUE TITLE
Feature Request - be consistent with part identifier

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#8008]
**Is your feature request related to a problem? Please describe.**

All pending some resolution of https://github.com/ArctosDB/arctos/discussions/5231, which at this point does not seem likely:

part ID is used for various things, and has an inconsistent name and format. Things should be made consistent and global.

* the internal identifier (int) should not be exposed
* everything that provides or accepts "partID" should use the format of https://arctos.database.museum/guid/OGL:Genomic:30366/PID32213975
* The name also isn't consistent, I'll call that "partID" to make locating it easy (eg so no chance of confusing with ``loan_item.part_id``) 
* EDIT: and going with parentPartID as a name for ``core.specimen_part.sampled_from_obj_id``

**Describe what you're trying to accomplish** 

Consistency and scalability.

**Describe the solution you'd like**

see above

**Describe alternatives you've considered**

fragmented tools with limited scope

**Additional context**

nope

**Priority**

Probably fairly high, tools are not interoperable and debugging isn't as straightforward as it should be, but there are no directly-relevant complaints either.

I think the only possible negative impact would involve data stored in loaders, and fixing that should be trivial.

Going next task, I know how to do this.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
How will this affect loading parts to containers using "part_id"? That is critical for my workflow to get inventory uploaded.

**Comment by dustymc** [comment]
> loading parts to containers using "part_id"

See "data stored in loaders" - 

* A file made pre-migration won't work post-migration, but I can upgrade it (or provide the formula)
* A file made (via any of the 'download....' or 'loader...' options) post-migration will work as always, although the key and values will be a bit different.

I don't think you'll notice, but please let me know if you have something outside of that framework.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
I'm not sure I understand. Basically what I need to continue doing is downloading part ID from view/download parts, and copying that over to my bulkload parts to containers spreadsheet.

**Comment by dustymc** [comment]
>continue

no problem

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Ok, thanks!
